modules/persistent-storage-local-install.adoc

```diff
@@ -28,12 +28,12 @@ To allow local storage creation on master and infrastructure nodes, add a tolera
 +
 [source,terminal]
 ----
-$ oc patch ds local-storage-local-diskmaker -n local-storage -p '{"spec": {"template": {"spec": {"tolerations":[{"operator": "Exists"}]}}}}'
+$ oc patch ds openshift-local-storage-local-diskmaker -n openshift-local-storage -p '{"spec": {"template": {"spec": {"tolerations":[{"operator": "Exists"}]}}}}'
```

Comment:
we should never ever patch daemonsets directly but anyways name of daemonsets are still `local-storage-local-diskmaker`, they have not changed.

Reply:
OK - I've dropped the `openshift-` prefix from these daemonset commands. But leaving comment open for QE to review.